class/Defaults/Carrier/Webhook.php

```diff
@@ -148,67 +148,28 @@ public function send( Triggerable $trigger ) {
 			$filtered_args = apply_filters_deprecated( 'notification/webhook/args/' . $url['type'], [ $args, $this, $trigger ], '6.0.0', 'notification/carrier/webhook/args/' . $url['type'] );
 			$filtered_args = apply_filters( 'notification/carrier/webhook/args/' . $url['type'], $filtered_args, $this, $trigger );
 
-			if ( 'get' === $url['type'] ) {
-				$this->send_get( $url['recipient'], $filtered_args, $headers );
-			} elseif ( 'post' === $url['type'] ) {
-				$this->send_post( $url['recipient'], $filtered_args, $headers );
-			}
+			$this->http_request( $url['recipient'], $filtered_args, $headers, $url['type'] );
 		}
 
 	}
 
 	/**
-	 * Sends GET request
-	 *
-	 * @since  5.0.0
-	 * @param  string $url  URL to call.
-	 * @param  array  $args    arguments.
-	 * @param  array  $headers headers.
-	 * @return void
-	 */
-	public function send_get( $url, $args = [], $headers = [] ) {
-
-		$remote_url = add_query_arg( $args, $url );
-
-		$remote_args = apply_filters_deprecated( 'notification/webhook/remote_args/get', [
-			[
-				'headers' => $headers,
-			],
-			$url,
-			$args,
-			$this,
-		], '6.0.0', 'notification/carrier/webhook/remote_args/get' );
-
-		$remote_args = apply_filters( 'notification/carrier/webhook/remote_args/get', $remote_args, $url, $args, $this );
-
-		$response = wp_remote_get( $remote_url, $remote_args );
-
-		do_action_deprecated( 'notification/webhook/called/get', [
-			$response,
-			$url,
-			$args,
-			$remote_args,
-			$this,
-		], '6.0.0', 'notification/carrier/webhook/called/get' );
-		do_action( 'notification/carrier/webhook/called/get', $response, $url, $args, $remote_args, $this );
-
-	}
-
-	/**
-	 * Sends POST request
+	 * Makes http request
 	 *
-	 * @since  5.0.0
+	 * @since  [Next]
 	 * @param  string $url  URL to call.
 	 * @param  array  $args    arguments.
 	 * @param  array  $headers headers.
+	 * @param string $method Http request method.
 	 * @return void
 	 */
-	public function send_post( $url, $args = [], $headers = [] ) {
+	public function http_request( $url, $args = [], $headers = [], $method ) {
 
 		$remote_args = apply_filters_deprecated( 'notification/webhook/remote_args/post', [
```

Comment:
```suggestion
		$remote_args = apply_filters_deprecated( 'notification/webhook/remote_args/' . $method, [
```

We had this for `get` as well, but now you left filter only for `post`